prody/proteins/pdbfile.py

```diff
@@ -1172,9 +1182,9 @@ def writePDBStream(stream, atoms, csets=None, **kwargs):
         if multi:
             write('MODEL{0:9d}\n'.format(m+1))
         for i, xyz in enumerate(coords):
-            if i == 99999:
+            if pdbline != PDBLINE_GE100K and (i == 99999 or serials[i] >= 10000):
                 pdbline = PDBLINE_GE100K
```

Comment:
Raise a warning here saying the number of atoms is too big.

Reply:
This one likewise.